src/Controller/UserController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\RegisterType;
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class UserController extends AbstractController
+{
+    /**
+     * @Route("/registration")
+     */
+    public function registerAction(Request $request, UserPasswordEncoderInterface $passwordEncoder)
+    {
+        $user = new User();
+        $form = $this->createForm(RegisterType::class, $user);
+
+        $form->handleRequest($request);
```

Comment:
Form cannot be valid if it is not submitted, so submitted check is not necessary